### ISSUE TITLE
Unnatural jumping of page when scrolling.

### ISSUE THREAD

**Issue by sandydo** [nextcloud/nextcloud.com#382]
Hey guys,

there is a problem with the head menu bar of the nextcloud site. It seems that only in the default position the menu bar realizes being above the page. When you start scrolling then it flips just behind the bar.

This means for example on the first page the words "Get Started" are only visible in the default position. If you start scrolling it jumps behind the menu bar.

Cheers,
Thomas

**Comment by MariusBluem** [comment]
https://github.com/nextcloud/nextcloud.com/issues/297

@jospoortvliet @Espina2

**Comment by Espina2** [comment]
@sandydo 
Did you recheck this with clean cache? What browser and you are using and OS?

**Comment by Espina2** [comment]
Please post a print screen of the issue to.

**Comment by sandydo** [comment]
Ok, a bit more detail:

The Problem occurs with Chrome, Firefox and Safari on Mac OS!

For you to reconstruct it:
- Go to  https://nextcloud.com/about/
- Scroll a minimal amount the part with the main information.
- Watch how the "About" jumps behind the menu bar.

I added two pictures. One with the loaded page just without any movement and the other when scrolling just a bit (one pixel of scrolling should be enough).
<img width="1280" alt="bildschirmfoto 2017-01-30 um 22 17 15" src="https://cloud.githubusercontent.com/assets/5897058/22442025/14332fde-e73a-11e6-8e2e-dd27f7813162.png">
<img width="1280" alt="bildschirmfoto 2017-01-30 um 22 17 20" src="https://cloud.githubusercontent.com/assets/5897058/22442026/1435755a-e73a-11e6-8f2b-e83246b38c3f.png">


Cheers

**Comment by Espina2** [comment]
I already figure it out. Thanks for the all the details, I will fix this.

**Comment by jancborchardt** [comment]
Yeah, this is especially easy to see on the pages of Providers or News. The header reappears very janky and turns white, then blue.

@Espina2 @nextcloud/designers we should always keep the header blue. The switch to white is just weird and I just had a friend who used the site mention it to me again as if it was a bug.

**Comment by Espina2** [comment]
@jancborchardt the default header is transparent, we make it to work like this for the old legacy pages. When we migrate to the new language this it will not happen.The legacy pages share the main.js file so this is are happening. 

I already migrate a lot of them so its a matter of time until this are fixed in the whole website.

**Comment by Espina2** [comment]
Ok this still happens on the old pages. :)
